felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/Verge.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.kontrakter.felles.tilbakekreving
+
+import no.nav.familie.kontrakter.felles.PersonIdent
+import org.springframework.format.annotation.DateTimeFormat
+import java.time.LocalDateTime
+
+data class Verge(val vergeType: VergeType,
+                 @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
```

Comment:
Unødvendig annotasjon dette er standard for LocalDateTime

Reply:
fjernet